### ISSUE TITLE
Feature:Added Floating action button menu for task selection

### ISSUE THREAD

**Issue by icancode23** [scorelab/kute#65]
**Feature:**

Instead of an activity of task selection,implemented a FAB menu which can enhance user experience

**Snapshots:**

![ezgif com-video-to-gif](https://cloud.githubusercontent.com/assets/20622168/25248673/baf5141e-262c-11e7-9055-a055712bd1e1.gif)

**Implementation**

Can submit a PR for review

**Comment by CloudyPadmal** [comment]
How about a transition implementation like this one? :)

![rfab_label_list](https://cloud.githubusercontent.com/assets/14261304/25263527/f1bd4222-267d-11e7-9dd0-dfb7bca5124b.gif)

**Comment by icancode23** [comment]
@CloudyPadmal yes brother we can implement it in this way as well but just that the implementation i have mentioned above is a bit more clear for the user this is what i think.(again the same point that opinions may differ)

**Comment by CloudyPadmal** [comment]
Agreed!! :+1:
